WPMedia/fusion-news-theme-blocks
# THEMES-757: Identity Blocks | Header action block

## Description

This PR adds the base files for the Identity blocks, as well as the v2 converted Identity Header Account Block.

## Jira Ticket

- [THEMES-757](https://arcpublishing.atlassian.net/browse/THEMES-757)

## Acceptance Criteria

Identity Header Account Block is converted for Themes v2

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-757`
2. Checkout the feature pack branch `git checkout THEMES-757-2` 
    - In the `blocks.json` change `"@wpmedia/identity-block",` to `"@wpmedia/identity-block@local",` to avoid issues pulling in the old identity block code.
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block`
4. In the header nav chain, add the Identity Header Account - Arc Block as part of the chain, and set it to display on the right on both desktop and mobile.
    - ![Screenshot 2023-07-06 at 9 59 05 AM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/91c0dc46-aaaa-42e6-8e9e-a70b412369cd)

    - ![Screenshot 2023-07-06 at 9 58 45 AM](https://github.com/WPMedia/arc-themes-blocks/assets/4759882/16775a4e-df03-435e-bcfc-dd914c60b004)

5. Ensure the styling matches what is expected for v2 buttons. They can be compared to the section and search buttons also in the header nav.
6. At mobile sizes, the button should open a dropdown showing the sign up and login links.

## Dependencies or Side Effects

- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/433

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-757]: https://arcpublishing.atlassian.net/browse/THEMES-757?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
